sharktank/sharktank/models/vae/layers.py

```diff
@@ -0,0 +1,261 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
```

Comment:
Just realized. This should probably be broken up and put in `sharktank/sharktank/layers/`

Reply:
Yeah, it should. I kept it separate following punet layers and was planning to clean up both in a follow up PR